main.go

```diff
@@ -172,6 +174,7 @@ func main() {
 		"labels-filter", labelsFilter,
 		"enable-multi-instrumentation", enableMultiInstrumentation,
 		"enable-apache-httpd-instrumentation", enableApacheHttpdInstrumentation,
+		"enable-dotnet-instrumentation", enableDotnetInstrumentation,
```

Comment:
Here also.

Reply:
done ✅ 
I've just updated all the references that I worked on